include/NovelRT/Ecs/Catalogue.h

```diff
@@ -0,0 +1,42 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_ECS_CATALOGUE_H
+#define NOVELRT_ECS_CATALOGUE_H
+
+#include "ComponentCache.h"
+#include "ComponentView.h"
+#include "EcsUtils.h"
+#include "EntityCache.h"
+
+namespace NovelRT::Ecs
+{
+    class Catalogue
+    {
+        private:
```

Comment:
IMO these labels should be at the same indentation level as the class, or the members indented one further.

Reply:
I wasn't sure what was agreed upon for this in terms of the new coding conventions so I just sort of "left it" for now. Do you want me to change this still?

Reply:
If the others are fine with it, I'll oblige - I'm not super sure what the new conventions are either if I'm honest :sweat_smile:

Reply:
I'm still working in that tbh @FiniteReality so I'll let this go as is - something I'll keep in mind for the clang-format / clang-tidy PRs :)